### PR TITLE
Refactored chip number logic

### PR DESCRIPTION
**Pull request by gary-wood**

### CODE REVIEW

File: app/src/js/main.js [hunk 1]
```diff
@@ -29,4 +29,37 @@ petDetectrApp.updateDateAndLocationText = function updateDateAndLocationText(e)
...
+petDetectrApp.setHideShow = function showHideElement(target, display) {
+    if (display) {
+        target.classList.remove('h-hide');
+    } else {
+        target.classList.add('h-hide');
+    }
+};
+
+// Chip number h-hide switching
+petDetectrApp.hideShowChipNumber = function hideShowChipNumber(e) {
```
Comment (DaveOrDead) [inline app/src/js/main.js:54]: Probably not important for this PR but wondering if there is a way we can make this more generic. So not specific to show hiding the chip number more like a function that toggles input visibility based on radio selection.

File: app/src/js/main.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ petDetectrApp.formSetup = function formSetup() {
     // Grab radio buttons and text nodes
     this.petStatus = document.querySelectorAll('[name="status"]');
     this.lastSeenText = document.querySelectorAll('.js-seen-found-text');
+    this.isChippedSelector = document.querySelectorAll('[name=is-chipped] + label');
```
Comment (DaveOrDead) [inline app/src/js/main.js:7]: We should think of a way of switching these selectors from [attributes] to JS- hooks. Not important for this PR

File: app/partials/pet-details-form.php [hunk 3]
```diff
@@ -138,7 +138,7 @@
         </p>
     </div>
 
-    <p class="h-spacing-base">
+    <p id="chip-number-wrap" class="h-spacing-base h-hide">
         <label for="chip-number">Chip number:</label>
```
Comment (DaveOrDead) [inline app/partials/pet-details-form.php:142]: I'm wondering if there are any ARIA implications on the radio where we link what it's opening similar to how you would with accessibility for a dropdown. I'm not 100% sure though

File: app/src/js/main.js (hunk 1 above)
Comment (gary-wood) [inline app/src/js/main.js:54]: Yep; I was thinking the same as I was building it.  Some sort of 'on' vs 'off' attribute/hook on the elements.  We just need to decide on the pattern

File: app/partials/pet-details-form.php (hunk 3 above)
Comment (DaveOrDead) [inline app/partials/pet-details-form.php:142]: or maybe we should just set focus into the text input once you've selected it?  That might be enough

Comment (gary-wood) [inline app/partials/pet-details-form.php:142]: Shifting focus to the input is a good shout - I'll implement that

**Comment by DaveOrDead** [comment]
LGTM

**Pull request merged**